Backpack/src/main/java/net/skyscanner/backpack/text/FontCache.kt

```diff
@@ -0,0 +1,24 @@
+package net.skyscanner.backpack.text
+
+import android.content.Context
+import android.graphics.Typeface
+import androidx.core.content.res.ResourcesCompat
+import java.util.Hashtable
+
+object FontCache {
+
+  private val fontCache = Hashtable<Int, Typeface>()
+
+  operator fun get(res: Int, context: Context): Typeface? {
+    var tf = fontCache[res]
+    if (tf == null) {
+      try {
+        tf = ResourcesCompat.getFont(context, res)
+      } catch (e: Exception) {
```

Comment:
Not sure about this, if for whatever reason this is not working "hiding" the exception like this will make it very hard to find the problem. I would remove it and let the exception be handled somewhere else, if the user puts a invalid font in the config we want it to fail and show the error no?